### ISSUE TITLE
[BUG] Scrollbar on Side Nav when Using Mouse

### ISSUE THREAD

**Issue by kpmurphh** [mobi/goponents#413]
**Describe the bug**
When the side navigation has enough links to expand for a scrollbar and a mouse is plugged in a scrollbar appears on the inside of the side navigation. See screenshot for reference. Can anything be done to hide this scrollbar when it's not being interacted with?

**To Reproduce**
1. Go to any side navigation that has a large number of links in the side navigation when using a mouse.

**Screenshots**
![MicrosoftTeams-image](https://user-images.githubusercontent.com/9440696/72754607-4b23ec00-3b96-11ea-8dd2-be2eecb1a684.png)

**Comment by grahamhency** [comment]
@kpmurphh this behavior is mostly driven by the browser, which is ideal. I would caution against messing with the appearance and behavior of the scroll bars typically. It does _have_ to be inside of the container because it is appearing because the container needs a scroll bar.

What behavior were you imaging this would be? Perhaps we can do something minor, but mostly this is browser driven.

**Comment by Fagiolia1** [comment]
@grahamhency as discussed yesterday, we took the code you provided Amit and modified it slightly:

```
::-webkit-scrollbar-track {     background-color: transparent;
}
::-webkit-scrollbar-thumb {     background-color: #b1b1b1;     border-radius: 6px;
}
::-webkit-scrollbar {     width: 8px;     height: 8px;
}
```
This gives the sidebar for the navigation as well as the table navigation a slightly cleaner look rather than the browser default that shows some harsh edges on the left navigation when open.

**Comment by grahamhency** [comment]
Closed with #448
